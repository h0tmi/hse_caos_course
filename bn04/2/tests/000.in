12
qwetyuqweuqwuer
asdfhashdfasdhf
zxcbvzbxcvbzxcrvb
asdhfashdfasdh
zxcbvzxbcvbzxcvq
asdfasdfasdf
zxcvzxcvzxcv
fasfasfasdef
tqtqeerqwerqwe
zxcvzxvczxcv
wefqwefqwef
czxcvzxcvzxc
