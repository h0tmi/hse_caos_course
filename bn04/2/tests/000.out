asdfasdfasdf
asdfhashdfasdhf
asdhfashdfasdh
czxcvzxcvzxc
fasfasfasdef
qwetyuqweuqwuer
tqtqeerqwerqwe
wefqwefqwef
zxcbvzbxcvbzxcrvb
zxcbvzxbcvbzxcvq
zxcvzxcvzxcv
zxcvzxvczxcv
